client/src/pages/SignUp.js

```diff
@@ -0,0 +1,192 @@
+    } else {
+      // Implement API call to send data to flask backend :: reminder
+      alert(
+        `name: ${form.name}\n email: ${form.email} \n password: ${form.password}\n re-password: ${form.password}\n experience: ${form.experience}`
+      );
+    }
+  };
+  return (
+    <>
+      <form action="" method="POST" autoComplete="off" onSubmit={handleLogin}>
+        {/* First step of SignUp */}
+        {step === "first" ? (
+          <Grid container direction="column" className={classes.container}>
+            <Typography
+              className={classes.mainHeading}
+              variant="h4"
+              align="center"
+              display="block"
+            >
+              Create an account
+            </Typography>
+            <FormControl>
+              <TextField
+                name="name"
+                type="text"
+                variant="outlined"
+                placeholder={"Name"}
+                className={classes.input}
+                value={form.name}
+                onChange={handleFormInput}
+                required
+              />
+              <TextField
+                name="email"
+                type="email"
+                variant="outlined"
+                placeholder={"E-mail address"}
+                className={classes.input}
+                value={form.email}
+                onChange={handleFormInput}
+                required
+              />
+              <TextField
+                name="password"
+                variant="outlined"
+                placeholder={"Password"}
+                className={classes.input}
+                type="password"
+                value={form.password}
+                onChange={handleFormInput}
+                required
+                inputProps={{ minLength: 6 }}
+                error={error}
+                helperText={error && "Password need to be matched !"}
+              />
+              <TextField
+                name="confirmPassword"
+                variant="outlined"
+                placeholder={"Confirm Password"}
+                className={classes.input}
+                type="password"
+                value={form.confirmPassword}
+                onChange={handleFormInput}
+                required
+                inputProps={{ minLength: 6 }}
+                error={error}
+                helperText={error && "Password need to be matched !"}
+              />
+            </FormControl>
+            <Button
+              variant="contained"
+              className={classes.loginButon}
+              type="submit"
+            >
+              Continue
+            </Button>
+            <Typography variant="body1" className={classes.bottomQuestion}>
+              Already have an account? <Link href="/signin">Login</Link>
+            </Typography>
+          </Grid>
+        ) : (
+          // Second Step of SignUp
+          <Grid container direction="column" className={classes.container}>
```

Comment:
This second step of Sign Up should be a separate page and it'll also be something that @MokhammadDaniyal will be working on as issue #12

Can you please remove this from this component?

In addition, we should be using React Router to show that onboarding page, as it'll be a lot easier and less messy to route to it in the future

Reply:
I think that page is different than this page info.

`. Page that shows that you can add your experience level for different languages`

but this step is for adding general experience level I guess and mock images have similar concept.

but anyways, I am removing the code. 👍

Reply:
Yeah, the mocks have a similar concept, we don't need this page 👍